tutorials/custom-email-domain-icloud/index.mdx

```diff
@@ -0,0 +1,125 @@
+
+<Message type="note">
+  While we provide examples to help understand where each value goes, we strongly recommend copying exactly the value provided by iCloud, including punctuations and/or symbols.
+</Message>
+
+### MX records
+
+There are two MX records. The correspondence between iCloud's MX instructions and Scaleway DNS records is as follows:
+
+| iCloud       | Scaleway DNS        | Values (example)                              |
+| ------------ | ------------------- | --------------------------------------------- |
+| Type         | Type                | MX                                            |
+| Host         | Name                | @                                             |
+| Value        | Hostname            | mx02.mail.icloud.com. / mx02.mail.icloud.com. |
+| Priority     | Priority            | 10                                            |
+
+<Lightbox src="scaleway-icloud-dns-mx.webp" alt="" />
+
+### TXT record
+
+The correspondence between iCloud's TXT instructions and Scaleway DNS records is as follows:
+
+| iCloud       | Scaleway DNS        | Values (example) |
+| ------------ | ------------------- | -----------------|
+| Type         | Type                | TXT              |
+| Host         | Name                | @                |
+| Value        | Value               | apple-domain=xxx |
+
+<Lightbox src="scaleway-icloud-dns-txt.webp" alt="" />
+
+### SPF record
+
+The correspondence between iCloud's SPF instructions and Scaleway DNS records is as follows:
+
+| iCloud       | Scaleway DNS        | Values (example)                 |
+| ------------ | ------------------- | -------------------------------- |
+| Type         | Type                | TXT                              |
+| Host         | Name                | @                                |
+| Value        | Value               | "v=spf1 include:icloud.com ~all" |
+
+<Lightbox src="scaleway-icloud-dns-spf.webp" alt="" />
+
+### DKIM record
+
+The correspondence between iCloud's DKIM instruction and Scaleway DNS records is as follows:
+
+| iCloud       | Scaleway DNS        | Values (example)                      |
+| ------------ | ------------------- | --------------------------------------|
+| Type         | Type                | CNAME                                 |
+| Host         | Name                | sig1._domainkey                       |
+| Value        | Hostname            | sig1.dkim.xxx.at.icloudmailadmin.com. |
+
+<Lightbox src="scaleway-icloud-dns-dkim.webp" alt="" />
+
+
+## Verifying domain's records on iCloud
+
+1. Go back to your iCloud account.
+2. Select **Finish setup** at step 4. Click **Confirm**.
+    <Lightbox src="scaleway-icloud-finish-setup.webp" alt="" />
+
+    <Message type="note">
+      It may take some time for the records to propagate across the network. You can try the **Finish setup** step again later. You can also use the `dig` command in your terminal to check if the domain's records are accessible.
```

Comment:
```suggestion
      It may take some time for the records to propagate across the network. You can try the **Finish setup** step again later. Refer to the [dedicated documentation](/domains-and-dns/how-to/test-dns-zones-with-dig/#how-to-query-a-specific-record-type) to check if the domain's records are accessible using the `dig` command.
```